rustworkx/digraph.pyi

```diff
@@ -13,7 +13,7 @@ import numpy as np
 from .iterators import *
 from .graph import PyGraph
 
-from typing import Any, Callable, Dict, Generic, TypeVar, Optional, List, Tuple, Sequence
```

Comment:
Unfortunately this is 3.9+ only and our stubs support Python 3.8. Is it hard to disable the Ruff warning for those?

Reply:
Is it? I have the pyupgrade rules set to target 3.8 and this was flagged by those rules. It also passed all the CI tests.

Reply:
It is because of [PEP 585](https://peps.python.org/pep-0585/). But we have the future import so it doesn’t matter!